seata-spring-autoconfigure/seata-spring-autoconfigure-core/src/main/java/io/seata/spring/boot/autoconfigure/properties/registry/RegistryNacosProperties.java

```diff
@@ -114,4 +115,15 @@ public RegistryNacosProperties setSecretKey(String secretKey) {
         this.secretKey = secretKey;
         return this;
     }
+
+    public String getPattern() {
```

Comment:
how to use this configuration item，should also add in:
https://github.com/seata/seata/tree/develop/script/client/conf
https://github.com/seata/seata/tree/develop/script/client/spring
https://github.com/seata/seata/blob/develop/script/config-center/config.txt

Reply:
> 

this config can be auto added as it's part of nacos @ConfigurationProperties